rpc/src/rpc_pubsub_service.rs

```diff
@@ -39,10 +40,17 @@ pub struct PubSubConfig {
 
 impl Default for PubSubConfig {
     fn default() -> Self {
+        let queue_capacity_items = if cfg!(test) {
```

Comment:
Oh. This is the issue. `cfg!(test)` doesn't propagate across crates, and `RpcSubscriptions::new()` is used in quite a few solana-core module unit tests.
Although it will be a lot of churn, `RpcSubscriptions::new_for_tests()` might be the best way to go. I'm happy to help with this, if you give me permission to push to your branch.

Reply:
https://github.com/CriesofCarrots/solana/tree/turbo-ws1

Reply:
Thank you for the help! Are we now good to merge?

Reply:
Yep, let's pull the trigger